src/main/java/com/oath/halodb/HaloDBInternal.java

```diff
@@ -110,13 +111,21 @@ static HaloDBInternal open(File directory, HaloDBOptions options) throws HaloDBE
             options.getFixedKeySize(), options.getMemoryPoolChunkSize()
         );
 
-        dbInternal.buildInMemoryIndex(options);
+        long maxSequenceNumber = dbInternal.buildInMemoryIndex(options);
+
```

Comment:
minor change: could you please remove line empty line?

Reply:
done